.github/workflows/open-pr.yml

```diff
@@ -14,13 +14,23 @@ env:
   SCORE_HELM_VERSION: 'latest'
   SCORE_HUMANITEC_VERSION: 'latest'
   WORKLOAD_NAME: my-sample-app
-  HUMCTL_VERSION: '0.15.0'
+  HUMCTL_VERSION: '0.19.1'
 jobs:
   build-run-test-push:
     runs-on: ubuntu-latest
     steps:
       - name: checkout code
         uses: actions/checkout@v4
+      - name: install humctl
+        uses: humanitec/setup-cli-action@v1
+        with:
+          version: ${{ env.HUMCTL_VERSION }}
+      - name: humctl score validate
+        run: |
+          humctl score validate score/score.yaml \
+              --token ${{ secrets.HUMANITEC_TOKEN }} \
+              --org ${{ secrets.HUMANITEC_ORG }} \
```

Comment:
Idea: You can set up all HUMANITEC_* variables as an environments vars, then you don't have to set it in each `humctl` step